news/2021-05-13-mappers-report-april.md

```diff
@@ -0,0 +1,172 @@
+
+Appearing as the runner-up, [Mao](https://osu.ppy.sh/users/2204515), on the other hand, not only achieves the third tier profile badge but also attains the *Elite Mapper* user title, which later turns into *Elite Mapper II* thanks to his participation in the [A Labour of Love](/wiki/Contests/A_Labour_of_Love_Contest) winning entry. As for the Screener Crew #1, [Acylica](https://osu.ppy.sh/users/1943309) takes the spotlight and getting an inch closer to the *Elite Mapper* title. Huge applause to them for their flawless victories!
+
+osu!taiko section of the contest had [Ph0eNiiXZ](https://osu.ppy.sh/users/9463721) rise forth to the first place, followed by [Nifty](https://osu.ppy.sh/users/4956097) as the runner-up. Many congrats to both of them!
+
+If you're craving for some details, [this newspost](https://osu.ppy.sh/home/news/2021-04-23-results-monthly-beatmapping-contest-march-2021) should satisfy your needs.
+
+Next up, the April edition of Monthly Beatmapping Contest greets us with a few novelties. For the first time ever, the contest will entertain osu!catch submissions in addition to osu! and osu!taiko. Though the submission period has since ended, the limitations for April were as follows:
+
+- **osu!**:
+    - All sliders must use red anchors.
+    - Red anchors must form clear corners in the slidershape.
+- **osu!taiko**:
+    - No more than 3 of the same note colours in a row.
+- **osu!catch**:
+    - Sliders cannot be used.
+
+If you're excited to see the results, we suggest you take few minutes to look through [this newspost](https://osu.ppy.sh/home/news/2021-04-08-monthly-beatmapping-contest-april-2021) to keep yourself up-to-date with the details.
+
+For those unfamiliar with the format of the Monthly Beatmapping Contest and its proceedings, go ahead and have a read of the [wiki article](/wiki/Contests/Monthly_Beatmapping_Contest).
+
+### osu! Beatmapping World Championship 2021: Registrations Open
+
+It's that time of the year again! As for the third mapping contest of this issue, we're featuring the third iteration of the osu! Beatmapping Championship, registrations for which have recently opened.
+
+Keeping the notion of country-based competition, the 2021 edition of the contest is allowing multiple teams to be formed to represent a particular country, yet only the best one among them will get the chance of proceeding further than the first round. May the best team win!
+
+What's more, a judging panel featuring a notable batch of professional player circle-clickers has been added to the contest proceedings, which will function like a side dish to the main contest. As a hit squad of high-level mappers are evaluating the submissions, the professional player panel will be testplaying these to rate them in accordance with a certain set of criteria. Sounds interesting, doesn't it?
```

Comment:
[Playtesting](https://en.wikipedia.org/wiki/Playtest) seems to be a real term, but test-playing needs a hyphen I believe

```suggestion
What's more, a judging panel featuring a notable batch of professional circle-clickers has been added to the contest proceedings, which will function like a side dish to the main contest. While a hit squad of high-level mappers are evaluating the submissions, the professional player panel will be test-playing these to rate them in accordance with a certain set of criteria. Sounds interesting, doesn't it?
```